### ISSUE TITLE
We should not expose docker registry and image name for OCP in CDK

### ISSUE THREAD

**Issue by LalatenduMohanty** [projectatomic/adb-utils#188]
We should not expose docker registry and image name for OCP in CDK in the rhel-ose Vagrantfile as these are constant for CDK.

example (at present):

```
DOCKER_REGISTRY="registry.access.redhat.com"
IMAGE_NAME="openshift3/ose"
IMAGE_TAG="v3.3.0.32"
```

IMO it should be just the image tag which will help users to change if they want to go to another version.
For example the Vagrrantfile should only contain 

```
IMAGE_TAG="v3.3.0.32"
```

**Comment by LalatenduMohanty** [comment]
@praveenkumar @budhrg @hferentschik comments?

**Comment by praveenkumar** [comment]
@LalatenduMohanty Agree, please check follow https://github.com/projectatomic/adb-utils/issues/190 also.
